Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -716,8 +737,9 @@ def get_categories_command(display_url):
     return entry
 
 
-def get_categories():
-    cmd_url = '/urlCategories'
+def get_categories(custom_only=False):
+    cmd_url = '/urlCategories?customOnly=true' if custom_only else '/urlCategories'
```

Comment:
`customOnly=true` should be passed as `params` in the HTTP request and not as part of the URL.
let the requests module do his job